src/utils/normalizeModuleFederationOptions.ts

```diff
@@ -313,10 +319,7 @@ export function getNormalizeModuleFederationOptions() {
 
 export function getNormalizeShareItem(key: string) {
   const options = getNormalizeModuleFederationOptions();
-  const shareItem =
-    options.shared[removePathFromNpmPackage(key)] ||
-    options.shared[removePathFromNpmPackage(key) + '/'];
-  return shareItem;
+  return options.shared[key];
```

Comment:
`removePathFromNpmPackage()` doesn't looks to be applied when creating the shared object.